app/helpers/application_helper.rb

```diff
@@ -1,2 +1,37 @@
 module ApplicationHelper
+  # This method makes it easier to yield content with a default value.
+  #
+  # USAGE:
+  # ```
+  # <% yield_default :signature do %>
+  #   Default signature here
+  # <% end %>
+  # ```
+  #
+  # The same functionality can be achieved with the following code:
+  # ```
+  # <% if content_for?(:signature) %>
+  #   <%= yield :signature %>
+  # <% else %>
+  #   Default signature here
+  # <% end %>
+  # ```
+  #
+  # In helpers, `content_for` is used instead of `yield`.
+  #
+  # For more information regarding `yield` and `content_for`, see:
+  # - https://guides.rubyonrails.org/layouts_and_rendering.html#understanding-yield
+  # - https://apidock.com/rails/ActionView/Helpers/CaptureHelper/content_for
+  # - https://apidock.com/rails/v5.2.3/ActionView/Helpers/CaptureHelper/content_for%3F
+  def yield_default(key, &block)
```

Comment:
`yield_default` seems kinda cryptic, maybe something like this?

```suggestion
  def default_for_or_yield_to(key, &block)
```

Reply:
[`31fc1cc` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/31fc1ccb33b9d2f1dc321b59b68f9e7d5ff3846b)